src/environments/environment.development.ts sample

```diff
@@ -0,0 +1,18 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```

Comment:
Would we just be able to remove this `suffix` and add this file to `.gitignore` after it's been committed? Right now, changes to this file are being tracked which could allow someone to accidentally push secrets

Reply:
Hmm.. I always thought this was the safe way to provide an env file template to share with others. Would it be safer if I gave explicit instructions in the README to rename the file before it's edited so that the changes are never tracked?

Reply:
You'll still be able to provide the template since it's committed to the repo. I think that renaming this file to what it needs to be saves the user an extra step since they're going to have to rename it anyways. Let me know if you wanna hop on a call to discuss more!